docs/content/setup/manual-setup.md

```diff
@@ -61,3 +62,64 @@ If you are upgrading HedgeDoc from an older version, follow these steps:
    This way it's easier to see warnings or errors that might occur.  
    <small>You can leave out `NODE_ENV=production` for development.</small>
 8. You can now restart the HedgeDoc server!
+
+## Systemd Unit Example
+Using the unit file below, you can run HedgeDoc as a systemd service.
+
+!!! warning
+    - In this example, you must configure HedgeDoc using the `config.json` file and the 
+    `production` key.
+    - Make sure the user and group `hedgedoc` exists and has appropriate permissions in the
+    directory you installed HedgeDoc in or change the `User` and `Group` settings in the unit
+    file.
+    - Make sure `WorkingDirectory` points to the directory you installed HedgeDoc in.
+    - Make sure `ReadWritePaths` contains all directories HedgeDoc might write to. This may
+    include the `public/uploads` folder if you configure local storage. If you use SQLite, you
+    must also include the directory where the database file is saved. **Do not save the SQLite
+    file in the root directory of the HedgeDoc installation**, but create a subfolder like `db`!
+
+```ini
+[Unit]
+Description=HedgeDoc - The best platform to write and share markdown.
+Documentation=https://docs.hedgedoc.org/
+After=network.target
```

Comment:
Shouldn't we add the database server here?

Reply:
added